tools/clang/test/CodeGenDXIL/hlsl/intrinsics/buffer-load-stores-scalars.hlsl

```diff
@@ -15,7 +17,7 @@
 // RUN: %dxc -DTYPE=double   -T vs_6_9 %s | FileCheck %s --check-prefixes=CHECK,F64
 
 // RUN: %dxc -DTYPE=float1    -T vs_6_9 %s | FileCheck %s
-// RUiN: %dxc -DTYPE=bool1     -T vs_6_9 %s | FileCheck %s --check-prefixes=CHECK,I1
```

Comment:
Double checking this is intentional?  I think I've seen some people use "RUiN" as a way to disable tests that don't work.  Re-enabling this test is probably beyond the scope of this change, especially if it introduces new issues.